manifests/init.pp

```diff
@@ -30,57 +53,93 @@
   $secret_key  = 'YOUR_KEY',
   $source      = undef,
   $manage_java = true,
+  $collector_host = undef,
+  $collector_port = undef,
+  $windows_temp_dir  = 'C:\users\Administrator\Downloads',
```

Comment:
why this dir? what if user is not administrator?

Reply:
@GalSvirin1 we agreed on  windows_temp_dir => 'C:\Windows\Temp', why did you change it ?

Reply:
This folder on Windows is protected, we may find ourselves in a permissions problem, BTW, this is not the `%TEMP%` folder (`%TEMP%` -> `C:\Users\%username%\AppData\Local\Temp`)

Reply:
> why this dir? what if user is not administrator?

This is default path of `downloads` folder, use may configure it as described in the example.